roles/credentials/tasks/template_github_credentials.yml

```diff
@@ -0,0 +1,22 @@
+---  
+- set_fact: 
+    social_auth_github_org_key: "{{ lookup('vars', item + '_social_auth_github_org_key' ) }}"
+    social_auth_github_org_secret: "{{ lookup('vars', item + '_social_auth_github_org_secret' ) }}"
+    
+- include_tasks: encrypt_credentials.yml
```

Comment:
@alanmoran A task named encrypt_credentials.yml doesn't appear to exist?

Reply:
@pmccarthy great catch. This must of changed name at some stage. In our repo it is still called encrypt_credentials.yml. It is now called encrypt_variables.yml.